src/app/components/client/Chart.tsx

```diff
@@ -14,16 +16,19 @@ import { Button } from "../server/Button";
 import { ModalOverlay } from "./dialog/ModalOverlay";
 import { Dialog } from "./dialog/Dialog";
 import ModalImage from "../client/assets/modal-default-img.svg";
-import Image from "next/image";
-import Link from "next/link";
+import { Scan } from "../../functions/server/onerep";
 
 export type Props = {
   data: Array<[string, number]>;
-  hasRunScan: boolean;
   isEligibleForFreeScan: boolean;
+  scanStatus?: Scan["status"];
 };
 
-export const DoughnutChart = (props: Props) => {
+export const DoughnutChart = ({
+  data,
```

Comment:
Is this a plugin you have that destructures all the props? Because if so, we probably want to standardise on that - it makes it harder to see the actual changes in the PR.

Reply:
No plugin. After reaching a certain threshold of props — usually more than two or three — I’m starting to destruct props rather than having to access `props` every time. That’s just personal preference and I get that this is adding noise. Reverted in [87f7f85](https://github.com/mozilla/blurts-server/pull/3431/commits/87f7f85f62d78fffee3da4603775f9331c27c872).

Reply:
Oh haha well, not sure if that was worth a full revert :sweat_smile: Another option for next time would be to make that change in a separate commit and call that out in the PR description, then I can just look at the other commits while reviewing. (Assuming there's not too many of them, otherwise it's a bit of a hassle.)